python/private/pypi/requirements.bzl.tmpl.workspace

```diff
@@ -61,12 +61,23 @@ def install_deps(**whl_library_kwargs):
     for name, requirement in _packages:
         group_name = requirement_group_mapping.get(name.replace("%%NAME%%_", ""))
         group_deps = all_requirement_groups.get(group_name, [])
-
-        whl_library(
-            name = name,
-            requirement = requirement,
-            group_name = group_name,
-            group_deps = group_deps,
-            annotation = _get_annotation(requirement),
-            **whl_config
-        )
+        if "@ file://" in requirement:
+          whl_file = whl_files[name]
+          whl_library(
```

Comment:
nit: the indentation needs to be fixed here.

Reply:
Done.